### ISSUE TITLE
Deployment uses FP32 TorchScript on CUDA — no FP16/TensorRT/ONNX path for ACT+DINOv2 policy on Jetson

### ISSUE THREAD

**Issue by matedev01** [Autodiscovery/Cerebro-Control#3]
Thanks for open-sourcing this — really nice work getting AVP teleop running on H1 in sports mode.

While reading the deployment path I noticed the trained policy is run as a plain FP32 TorchScript module on CUDA, which is likely leaving a lot of latency on the table for an onboard Jetson (Orin) target.

### What I see

In `scripts/deploy_sim.py`:

```python
# scripts/deploy_sim.py:42-46
def load_policy(policy_path, device):
    policy = torch.jit.load(policy_path, map_location=device)
    return policy
...
policy = load_policy(policy_path, device)
policy.cuda()
policy.eval()
```

and the per-step input/output:

```python
# scripts/deploy_sim.py:47-55, 122
image_data = torch.from_numpy(np.stack([left_img, right_img], axis=0)) / 255.0
image_data = image_data.view((1, 2, 3, 480, 640)).to(device='cuda')
...
output = policy(*data)[0].detach().cpu().numpy()
```

In `act/imitate_episodes.py::save_jit` the only export path is `torch.jit.trace` → `.pt`:

```python
# act/imitate_episodes.py:320-325
traced_policy = torch.jit.trace(policy, input_data)
save_path = os.path.join(exp_dir, f"traced_jit_{epoch}.pt")
traced_policy.save(save_path)
```

The backbone is DINOv2 ViT-S/14 (`act/detr/models/backbone.py:98-101, 137-138`), so each control tick does a full ViT forward + ACT transformer decode over two 480×640 RGB frames at the 30 Hz loop in `deploy_sim.py` (`dt=1/30`, `chunk_size=60` with temporal aggregation).

### Why this matters on Jetson

For an H1-class onboard compute (Jetson Orin AGX/NX), FP32 TorchScript on a DINOv2 backbone at 480×640×2 is usually well above the 33 ms control budget, even before temporal aggregation overhead. A few standard wins are not used today:

1. **No FP16/AMP path.** No `policy.half()` / `torch.autocast` in the deploy loop. On Orin's Tensor Cores FP16 alone is typically ~2× over FP32 for ViT-class backbones.
2. **No ONNX/TensorRT export.** `save_jit` only emits TorchScript. There's no `torch.onnx.export` and `requirements.txt` doesn't pull `onnx`, `onnxruntime-gpu`, `tensorrt`, or `torch-tensorrt`. A TRT FP16 engine for DINOv2-S + the ACT decoder typically lands another ~2–3× over PyTorch FP16, and INT8 with PTQ calibration is often another ~1.5–2× on top.
3. **No `torch.compile` fallback** for users who can't ship TRT.
4. **Per-step host sync.** `policy(*data)[0].detach().cpu().numpy()` forces a `cudaStreamSynchronize` every tick; pinned host buffers + async copy would hide some of that.
5. **CPU → GPU image transfer in the hot loop.** `torch.from_numpy(...).to('cuda')` runs each step without `pin_memory()` / `non_blocking=True`.

### Suggestions

- Add an FP16 deploy switch: `policy.half()` + cast inputs (or wrap the forward in `torch.autocast('cuda', dtype=torch.float16)`), and document parity vs. FP32 on a small replay set.
- Add an ONNX export alongside `save_jit` (the ACT + DINOv2 graph traces cleanly with fixed shapes 1×2×3×480×640 and 1×26).
- Add a TensorRT engine-build script (FP16 by default, INT8 with a calibration dataset of recorded `processed_episode_*.hdf5` frames) and a `TRTPolicy` wrapper with the same `(qpos, image) -> actions` signature so `deploy_sim.py` can swap runtimes by flag.
- Report measured p50/p99 inference latency on the Orin you ship with, against the 33 ms budget — happy to contribute a small benchmark harness if useful.

Would you be open to PRs along these lines? I can start with the ONNX export + FP16 path since those are the smallest changes and unblock the TRT work.
